tests/integration/regression_test.py

```diff
@@ -14,7 +14,7 @@ def test_443_handle_nonchunked_response_in_stream(self):
         with pytest.raises(docker.errors.APIError) as exc:
             for line in self.client.build(fileobj=dfile, tag="a/b/c"):
                 pass
-        assert exc.value.response.status_code == 500
+        assert exc.value.response.is_error
```

Comment:
should be a func call()

Reply:
specifically I think this should be `assert exc.value.is_error()`

Reply:
ah, dang; let me update 😅 - I'm really bad at python

Reply:
Updated; also happy to remove the new `is_error` if you think we should just inline it here